CHANGELOG.md

```diff
@@ -1014,6 +1014,7 @@
 activity and on its child transactions (which can be actuals, refunds, and adjustments)
 - Remove Auth0 user identifier from user management
 - Add sector and sector category codes to radio button text when adding a new activity
+- Fix an error where a new refund creation would silently fail when the value was non-numeric
```

Comment:
Updating would also silently fail if you tried to change the value to something non-numeric. (Also fixed by adding validation to the form object.)